packages/yoga/src/DropdownInput/web/DropdownInput.jsx

```diff
@@ -0,0 +1,262 @@
+import React, { useRef, useState } from 'react';
+import PropTypes from 'prop-types';
+import { theme, Box, Icon, Text } from '@gympass/yoga';
+import styled, { css } from 'styled-components';
+import { Check, ChevronUp, ChevronDown, Close } from '@gympass/yoga-icons';
+import { useOnClickOutside } from '../../hooks';
+
+const Wrapper = styled.div`
+  position: relative;
+`;
+
+const Container = styled.div`
+  display: flex;
+  border-radius: ${theme.radii.small}px;
+  justify-content: flex-start;
+  align-items: center;
+  background: #fff;
+  border: ${theme.borders.small}px solid #ccc;
+  box-sizing: border-box;
+  background: #fff;
+  height: 52px;
+  margin: ${theme.spacing.xxsmall}px 0 0;
+  transition: all 0.1s ease-in-out;
+  width: 320px;
+  ${({ showDropDown }) =>
+    showDropDown &&
+    css`
+      border-top: ${theme.borders.small}px solid #ccc;
+      border-left: ${theme.borders.small}px solid #ccc;
+      border-right: ${theme.borders.small}px solid #ccc;
+      border-radius: ${theme.radii.small}px ${theme.radii.small}px 0px 0px;
+      border-bottom: 0;
+    `}
+`;
+
+const Clear = styled.button`
+  background: none;
+  border: none;
+  cursor: pointer;
+  display: none;
+  outline: none;
+  padding: ${theme.spacing.medium}px;
+`;
+
+const Input = styled.input`
+  background: none;
+  border: none;
+  color: ${theme.colors.text.primary};
+  flex: 1;
+  font-family: ${({ theme: { yoga } }) => yoga.baseFont.family};
+  font-size: ${theme.fontSizes.small}px;
+  font-style: normal;
+  font-weight: ${theme.fontWeights.regular};
+  height: 100%;
+  letter-spacing: 0px;
+  line-height: ${theme.lineHeights.small}px;
+  outline: none;
+  text-align: left;
+  padding-left: ${theme.spacing.small}px;
+
+  &::placeholder {
+    color: ${theme.colors.text.secondary};
+  }
+
+  ${({ value }) =>
+    value &&
+    css`
+      + ${Clear} {
+        display: block;
+      }
+    `}
+`;
+
+const Divisor = styled.div`
+  width: 1px;
+  height: 28px;
+  background: ${theme.colors.elements.lineAndBorders};
+`;
+
+const ContainerDropDown = styled.ul`
+  margin: 0;
+  height: 272px;
+  width: ${props => `${props.width}px`};
+  overflow-y: scroll;
+  border-left: 1px solid #ccc;
+  border-right: 1px solid #ccc;
+  border-bottom: 1px solid #ccc;
+  list-style: none;
+  padding: 0 16px 0 16px;
+  transition: all 0.1s ease-in-out;
+  border-radius: 0px 0px ${theme.radii.small}px ${theme.radii.small}px;
+  background: #fff;
+  position: absolute;
+  top: ${props => `${props.position}px`};
+  z-index: 9999;
+`;
+
+const ButtonDropDown = styled.button`
+  display: flex;
+  align-items: center;
+  width: auto;
+  background: none;
+  border: none;
+  outline: none;
+  cursor: pointer;
+  padding: 0;
+  height: 100%;
+`;
+
+const ItemDropDown = styled.li`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  align-content: center;
+  width: 100%;
+  height: 72px;
+  cursor: pointer;
+  font-size: 16px;
```

Comment:
use a yoga sizes theming and spaces for that!

Reply:
✅